applications/asset_tracker_v2/src/events/location_module_event.h

```diff
@@ -0,0 +1,179 @@
+	 *  the ``data.neighbor_cells`` member.
+	 */
+	LOCATION_MODULE_EVT_NEIGHBOR_CELLS_DATA_READY,
+
+	/** The location search timed out without acquiring location.
+	 *  The event has associated payload of the type ``struct location_module_data`` in
+	 *  the event struct member ``data.location``.
+	 *  The ``data.location`` struct member ``search_time`` contains valid data
+	 *  and ``satellites_tracked`` contain valid data for GNSS.
+	 */
+	LOCATION_MODULE_EVT_TIMEOUT,
+
+	/** An active location search has started. */
+	LOCATION_MODULE_EVT_ACTIVE,
+
+	/** A location search has stopped, either as a result of timeout, error
+	 *  or acquired location.
+	 */
+	LOCATION_MODULE_EVT_INACTIVE,
+
+	/** The module has been shut down gracefully.
+	 *  The event has associated payload of the type ``uint32`` in the struct member
+	 *  ``data.id`` that contains the module ID used when acknowledging a shutdown
+	 *  request from the util module.
+	 */
+	LOCATION_MODULE_EVT_SHUTDOWN_READY,
+
+	/** The location library has reported that it needs GPS assistance data.
+	 *  The event has associated payload of the type ``struct nrf_modem_gnss_agps_data_frame``
+	 *  in the event struct member ``data.agps_request``, which contains the types
+	 *  of A-GPS data that the modem needs.
+	 */
+	LOCATION_MODULE_EVT_AGPS_NEEDED,
+
+	/** The location library has reported that it needs GPS prediction data.
+	 *  The event has associated payload of the type ``struct gps_pgps_request``
+	 *  in the event struct member ``data.pgps_request``, which specifies the P-GPS data need.
+	 */
+	LOCATION_MODULE_EVT_PGPS_NEEDED,
+
+	/** An error has occurred, and data may have been lost.
+	 *  The event has associated payload of the type ``int`` in the struct member
+	 *  ``data.err``, which contains the original error code that triggered
+	 *  the sending of this event.
+	 */
+	LOCATION_MODULE_EVT_ERROR_CODE,
+};
+
+/** @brief Position, velocity and time (PVT) data. */
+struct location_module_pvt {
+	/** Longitude in degrees. */
+	double longitude;
+
+	/** Latitude in degrees. */
+	double latitude;
+
+	/** Altitude above WGS-84 ellipsoid in meters. */
+	float altitude;
+
+	/** Position accuracy (2D 1-sigma) in meters. */
+	float accuracy;
+
+	/** Horizontal speed in m/s. */
+	float speed;
+
+	/** Heading of user movement in degrees. */
+	float heading;
+};
+
+/** @brief Location module data for associated payload for event of
+ *         LOCATION_MODULE_EVT_NEIGHBOR_CELLS_DATA_READY types.
+ */
+struct location_module_neighbor_cells {
+	/** Information about the current cell. */
+	struct lte_lc_cells_info cell_data;
+	/** Information about the neighbor cells. */
+	struct lte_lc_ncell neighbor_cells[17];
+	/** Uptime when the event was sent. */
+	int64_t timestamp;
+};
+
+/** @brief Location module data for associated payload for events of LOCATION_MODULE_EVT_TIMEOUT and
+ *	   LOCATION_MODULE_EVT_GNSS_DATA_READY types.
+ */
+struct location_module_data {
```

Comment:
is this used only with gnss? if yes, it might be good to clarify that in naming. Here or/and in location_module_event